main/python/rootcp.py

```diff
@@ -32,13 +32,19 @@
 
 - rootcp -c 1 source.root:hist dest.root
   Change compression factor of 'dest.root' if not existing and copy the histogram named 'hist' from 'source.root' into it.
+
+- rootcp -a 5 9 source.root dest.root
```

Comment:
Why the choice of 'a' followed by 2 arguments?  Did you consider
```
rootcp -c 5 -l 9 source.root dest.root
```
(assuming `-l` is available).

Reply:
I wanted to clearly separate settings of algorithm and level. In option `-c` both are convoluted.

Nevertheless, it's no longer relevant given https://github.com/root-project/root/pull/14714#issuecomment-1952406831